content/posts/2020/02/libplanet-0.8.eng.md

```diff
@@ -0,0 +1,111 @@
+
+The built-in `IStore` implementation of `LiteDBStore` has been removed and [`DefaultStore`][DefaultStore] has replaced it.
+
+The reason for this change is that a single LiteDB file is no longer saved, but a directory that contains multiple formats.
+
+Another reason was our intention to remove implementation details from the name, with efforts to optimize the repository until the 1.0.0 release.
+
+In addition, a `compress` option has been created in the [`DefaultStore()` Creator][DefaultStore()] to reduce storage space. Although the option is still turned off by default in this version, the default will be change to `true` in the next version.
+
+[DefaultStore]: https://docs.libplanet.io/master/api/Libplanet.Store.DefaultStore.html
+[DefaultStore()]: https://docs.libplanet.io/master/api/Libplanet.Store.DefaultStore.html#Libplanet_Store_DefaultStore__ctor_System_String_System_Boolean_System_Boolean_System_Int32_System_Int32_System_Int32_System_Int32_System_Boolean_System_Boolean_
+
+
+[`ICryptoBackend`][ICryptoBackend]
+----------------------------------
+
+To be used on a variety of platforms, Libplanet has been using a cryptographic library written in pure C#, [Bouncy Castle]. Although pure C# implementations are a great advantage in portability, they serve as a penalty in performance.
+
+In the new version, an abstraction layer called [`ICryptoBackend`][ICryptoBackend] has been added to allow game developers to choose between portability and performance. While the default implementation, [`DefaultCryptoBackend`][DefaultCryptoBackend], is still internally dependent on Bouncy Castle, game developers can achieve performance benefits by implementing the `ICryptoBackend` interface appropriately, depending on the game’s target platform.
+
+For example, if you want Libplanet to use `MyCryptoBackend` class that implements `ICryptoBackend`, you can overwrite [`CryptoConfig.CryptoBackend` Attribute][CryptoConfig.CryptoBackend] as shown below.
+
+~~~~ csharp
+CryptoConfig.CryptoBackend = new MyCryptoBackend();
+~~~~
+
+In the case of [Nine Chronicles], a game currently being developed by our team, we have also improved game performance by implementing `ICryptoBackend`  interface that calls [secp256k1] C library unveiled in the Bitcoin project.
+
+[Bouncy Castle]: http://www.bouncycastle.org/csharp/
+[ICryptoBackend]: https://docs.libplanet.io/0.8.0/api/Libplanet.Crypto.ICryptoBackend.html
+[DefaultCryptoBackend]: https://docs.libplanet.io/0.8.0/api/Libplanet.Crypto.DefaultCryptoBackend.html
+[CryptoConfig.CryptoBackend]: https://docs.libplanet.io/0.8.0/api/Libplanet.Crypto.CryptoConfig.html#Libplanet_Crypto_CryptoConfig_CryptoBackend
+[secp256k1]: https://github.com/bitcoin-core/secp256k1
+
+
+Routing Table Improvement
+-------------------------
+
+Because Libplanet uses [Distributed Hash Table method][DHT] to communicate with other peers, it stores information about the peers associated with itself on the routing table.
```

Comment:
```suggestion
Because Libplanet uses [<abbr title="distributed hash table">DHT</abbr>][DHT] to communicate with other peers, it stores information about the peers associated with itself on the routing table.
```